wbooks-api/spec/models/rent_spec.rb

```diff
@@ -1,10 +1,31 @@
 require 'rails_helper'
 
 describe Rent do
+
   it { should validate_presence_of(:loan) }
   it { should validate_presence_of(:restitution) }
+  it { should validate_presence_of(:user) }
+  it { should validate_presence_of(:book) }
 
   it do
     expect(build(:rent)).to be_valid
   end
+
+  describe '#create' do
+    context 'When the loan is greater than restitution date' do
+      it 'is not valid' do
+        expect(build(:rent, loan: "2018-09-21", restitution: "2018-09-18")).not_to be_valid
```

Comment:
If loan and restitution are date types, I'd prefer using faker for both and then calculating the previous/next dates. For example, you'd do `Faker::Date.between(2.days.ago, Date.today) + 1.day` or `Faker::Date.backward(14) - 5.minutes`